runtime/heiko/src/constants.rs

```diff
@@ -14,6 +14,9 @@
 
 pub mod currency {
     use primitives::Balance;
+
+    pub const EXISTENTIAL_DEPOSIT: u128 = 500;
+
```

Comment:
Can we remove empty line?

Reply:
> There are still unresolved questions, let's slow the merge process a bit.

I think this one doesn't have any security issue, it's just how shall we use it.